app/src/main/java/org/mozilla/webmaker/activity/Element.java

```diff
@@ -5,13 +5,13 @@
 import android.os.Environment;
 import android.util.Log;
 
-import java.io.File;
-import java.util.UUID;
-
 import org.mozilla.webmaker.R;
 import org.mozilla.webmaker.WebmakerActivity;
 import org.mozilla.webmaker.util.Image;
 
+import java.io.File;
```

Comment:
Is your IDE making all these style changes automatically? If so, can you please just commit the changes needed to land the feature?